python/ray/tune/trainable/session.py

```diff
@@ -204,7 +204,7 @@ def _shutdown():
     _session = None
 
 
-@PublicAPI
+@Deprecated(message=_deprecation_msg)
```

Comment:
just checking that this won't print out the message when the method is called right?
It is just reflecting in the doc string?

Reply:
yep thats right, this is just in the docstring (which actually won't really matter currently since we no longer show these APIs in the docs anymore).